### ISSUE TITLE
Location of Filesaver.js changed

### ISSUE THREAD

**Issue by buschtoens** [cogniteev/ember-cli-file-saver#12]
The upstream library changed the location of `Filesaver.js` in the patch release `1.3.4`. This breaks the build process, since the file cannot be found.

```
Build Error (Funnel)

ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'


Stack Trace and Error Report: /tmp/error.dump.968897dfac3e8322c3151ef25d835d5a.log
```

```
broccoliBuilderErrorStack: Error: ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
    at Object.fs.lstatSync (fs.js:965:11)
    at symlink (/home/jan/gitlab/web/frontend/app/node_modules/symlink-or-copy/index.js:99:26)
    at Function.symlinkOrCopySync [as sync] (/home/jan/gitlab/web/frontend/app/node_modules/symlink-or-copy/index.js:85:5)
    at Funnel._copy (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:494:19)
    at Funnel.processFile (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:477:8)
    at Funnel.applyPatch [as _applyPatch] (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:390:12)
    at Funnel.<anonymous> (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:344:10)
    at Array.forEach (<anonymous>)
    at Funnel.processFilters (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:343:11)
    at Funnel.build (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:259:10)
  - codeFrame: ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
  - errorMessage: Build Canceled: Broccoli Builder ran into an error with `Funnel` plugin. 💥
ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Build Canceled: Broccoli Builder ran into an error with `Funnel` plugin. 💥
ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
  - name: Error
  - nodeAnnotation: Funnel
  - nodeName: Funnel
  - originalErrorMessage: ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
  - stack: Error: ENOENT: no such file or directory, lstat '/home/jan/gitlab/web/frontend/app/tmp/funnel-input_base_path-XwgE6lH6.tmp/FileSaver.js'
    at Object.fs.lstatSync (fs.js:965:11)
    at symlink (/home/jan/gitlab/web/frontend/app/node_modules/symlink-or-copy/index.js:99:26)
    at Function.symlinkOrCopySync [as sync] (/home/jan/gitlab/web/frontend/app/node_modules/symlink-or-copy/index.js:85:5)
    at Funnel._copy (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:494:19)
    at Funnel.processFile (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:477:8)
    at Funnel.applyPatch [as _applyPatch] (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:390:12)
    at Funnel.<anonymous> (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:344:10)
    at Array.forEach (<anonymous>)
    at Funnel.processFilters (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:343:11)
    at Funnel.build (/home/jan/gitlab/web/frontend/app/node_modules/broccoli-funnel/index.js:259:10)
```

**Comment by buschtoens** [comment]
Locking to `1.3.3` fixes this temporarily.

**Comment by shcherbanikolay** [comment]
I have the same issue.

**Comment by panthony** [comment]
Thanks for reporting !

I enforced the `1.3.3` version (https://github.com/cogniteev/ember-cli-file-saver/commit/e748d132d179e5cf84e8c86ef3d74d02b1cc72cd) and published it as `1.2.1`.

**Comment by buschtoens** [comment]
Semi-related: https://github.com/eligrey/FileSaver.js/issues/414
